### ISSUE TITLE
[Bug] RangeSlider touch not working on WPF

### ISSUE THREAD

**Issue by RwSgxx** [xamarin/XamarinCommunityToolkit#1395]
### Steps to Reproduce

1. Crate a new Xamarin Forms App in VS.
2. Clear everything in the Main Page and add a RageSlider (set in, max...)
3. Add a WPF project for the Xamarin Forms App you just created
4. The slider will work on the other platforms but not in WPF
5. In WPF, I can't grab the slider point and drag them.

### Expected Behavior
-- in the reproduction steps

### Actual Behavior
-- in the reproduction steps

### Basic Information
-- in the reproduction steps

- Version with issue: 1.1.0
- Last known good version: ----
- IDE: VS
- Platform Target Frameworks: 
  - WPF: .Net Framework 4.72
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:

**Comment by AndreiMisiukevich** [comment]
Hi @RwSgxx, thanks for the report

Could you please check if PanGestureRecognizer works on WPF?

RangeSlider is cross-platform control and it based on PanGestureRecognizer. Perhaps there are some bugs with WPF implementation of this gesture.

**Comment by RwSgxx** [comment]
It is because of the PanGestureRecognizer. It is designed to work with touch events not mouse events.

**Comment by AndreiMisiukevich** [comment]
@RwSgxx, unfortunately, I don't think this will be implemented in near future then(

**Comment by RwSgxx** [comment]
@AndreiMisiukevich I know. The iOS build is broken since 3 weeks and the Xamarin Team and Microsoft don't give a flying ... (you know what).
https://developercommunity.visualstudio.com/t/xamarinforms-ios-ioexception-sharing-violation-on/1355811?from=email&viewtype=all#T-N1450332
Totally disappointed. I regret investing so much time in this platform.

**Comment by AndreiMisiukevich** [comment]
@RwSgxx sorry to hear this :(
But unfortunately, I can't help with that. I am even not an MS employee.

**Comment by jfversluis** [comment]
@RwSgxx this is not the place to share issues for other products. Also, there are multiple workarounds given and even a message in the link that you shared that it should be fixed: https://developercommunity.visualstudio.com/t/xamarinforms-ios-ioexception-sharing-violation-on/1355811#T-N1450827

**Comment by RwSgxx** [comment]
@jfversluis you're right.  The part with "don't think this will be implemented in near future then" hit a nerve :)

-------------------------------------------------------------
So back to the problem it self.
1. The PanGestureRecognizer for WPF is design to work with touch events, not mouse events -> On a touch screen it will work. 
2. You need to create a custom MouseDragGestureRecognizer for WPF, it should expect 3 inputs DragStarted, DragContinue, DragCompleted (each of them receiving also the mouse position).  This should expose the same event as the PanGestureRecognizer (PanUpdated) so that it can be easily integrated in the RangeSlider.
3. The last piece is a custom renderer for the ThumbFrame for WPF. This should attach to the MouseDown, MouseMove and MouseUp events of the native control and forward them to the MouseDragGestureRecognizer. Here don't forget to capture and release the mouse, on MouseDown and MouseUp.
4. In the RangeSlider add the MouseDragGestureRecognizer to the two Thumbs.

**Comment by AndreiMisiukevich** [comment]
@RwSgxx thanks for the suggestion)
Feel free to grab this part and submit a PR :) Since this toolkit has "Community" word in the name ;)
